packages/frontend/src/hooks/usePieChartConfig.ts

```diff
@@ -192,8 +195,20 @@ const usePieChartConfig: PieChartConfigFn = (
         if (isLoading || allNumericMetricIds.length === 0) return;
         if (metricId && allNumericMetricIds.includes(metricId)) return;
 
+        // If the metric is a table calculation, we need to find the new name
```

Comment:
Like this comment here - I like it but I would go further and explain why we have to find the new name - I think this should be added to every use...Config hook we changed - what do you think?